x/pylons/keeper/pylons_account.go

```diff
@@ -55,6 +55,22 @@ func (k Keeper) GetPylonsReferral(ctx sdk.Context, addr string) (val types.Refer
 	return val, true
 }
 
+func (k Keeper) SetPylonsKYC(ctx sdk.Context, kycaddress types.KYCAddress, username types.Username) {
+	binaryKYC := k.cdc.MustMarshal(&kycaddress)
+	kycPrefixStore := prefix.NewStore(ctx.KVStore(k.storeKey), types.KeyPrefix(types.KYCAddressKey))
+	kycPrefixStore.Set(types.KeyPrefix(kycaddress.Value), binaryKYC)
```

Comment:
do not link kyc address and username

Reply:
Fix in https://github.com/Pylons-tech/pylons/pull/1018/commits/e94ff34e5aebe8a54ee8a8f16ca6ce7a4c4e0b6c